debugging/book-library/script.js

```diff
@@ -25,79 +26,79 @@ const author = document.getElementById("author");
 const pages = document.getElementById("pages");
 const check = document.getElementById("check");
 
-//check the right input from forms and if its ok -> add the new book (object in array)
-//via Book function and start render function
+// Book constructor function
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+// Add a new book and render the table
 function submit() {
-  if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
-  ) {
+  if (!title.value.trim() || !author.value.trim() || !pages.value.trim()) {
     alert("Please fill all fields!");
-    return false;
-  } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
-    render();
+    return;
   }
-}
 
-function Book(title, author, pages, check) {
-  this.title = title;
-  this.author = author;
-  this.pages = pages;
-  this.check = check;
+  const newBook = new Book(
+    title.value.trim(),
+    author.value.trim(),
+    pages.value.trim(),
+    check.checked
+  );
+  myLibrary.push(newBook);
+  render();
+
+  // Clear input fields after submission
+  title.value = "";
+  author.value = "";
+  pages.value = "";
+  check.checked = false;
 }
 
+// Render the library table
 function render() {
-  let table = document.getElementById("display");
-  let rowsNumber = table.rows.length;
-  //delete old table
-  for (let n = rowsNumber - 1; n > 0; n-- {
+  const table = document.getElementById("display");
+  const rowsNumber = table.rows.length;
+
+  // Delete all rows except the header
+  for (let n = rowsNumber - 1; n > 0; n--) {
     table.deleteRow(n);
   }
-  //insert updated row and cells
-  let length = myLibrary.length;
-  for (let i = 0; i < length; i++) {
-    let row = table.insertRow(1);
-    let cell1 = row.insertCell(0);
-    let cell2 = row.insertCell(1);
-    let cell3 = row.insertCell(2);
-    let cell4 = row.insertCell(3);
-    let cell5 = row.insertCell(4);
-    cell1.innerHTML = myLibrary[i].title;
-    cell2.innerHTML = myLibrary[i].author;
-    cell3.innerHTML = myLibrary[i].pages;
 
-    //add and wait for action for read/unread button
-    let changeBut = document.createElement("button");
-    changeBut.id = i;
-    changeBut.className = "btn btn-success";
-    cell4.appendChild(changeBut);
-    let readStatus = "";
-    if (myLibrary[i].check == false) {
-      readStatus = "Yes";
-    } else {
-      readStatus = "No";
-    }
-    changeBut.innerHTML = readStatus;
+  // Populate the table with library books
+  myLibrary.forEach((book, i) => {
+    const row = table.insertRow(-1);
+    const cell1 = row.insertCell(0);
```

Comment:
(This is pre-existing code, not code you wrote): What do you think about the naming of these variables? Would you change them in any way if you were writing this code yourself?